src/i18n/index.ts

```diff
@@ -0,0 +1,7 @@
+import enLocale from "./locales/en.json";
+
+export type enIds = keyof typeof enLocale;
```

Comment:
Where are we using this type?

Reply:
I was exploring the ways to add type safety to the ids and forgot to remove this. will delete.